pipelines/models/tensorflow-housing/README.md

```diff
@@ -0,0 +1,27 @@
+# How was this model generated?
+
+## Data source
+
+The model uses
+https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_california_housing.html#sklearn.datasets.fetch_california_housing,
+see
+https://inria.github.io/scikit-learn-mooc/python_scripts/datasets_california_housing.html
+for more information about the data.
```

Comment:
```suggestion
The model uses the [scikit-learn California Housing Dataset](https://scikit-learn.org/stable/datasets/real_world.html#california-housing-dataset), see [this course notebook](https://inria.github.io/scikit-learn-mooc/python_scripts/datasets_california_housing.html) for more information about the data.
```

Reply:
Thanks, updated in 8d26a2d4503e905be17c7548045bca67da6a905e.